libtsuba/include/katana/tsuba.h

```diff
@@ -57,6 +58,9 @@ OpenFlagsValid(uint32_t flags) {
 KATANA_EXPORT katana::Result<RDGManifest> FindManifest(
     const std::string& rdg_name);
 
+KATANA_EXPORT katana::Result<katana::RDGManifest> FindManifest(
```

Comment:
Is this finding the latest manifest? Maybe that should be in the name.

Reply:
It's finding manifest first by checking the cache in the `TxnContext`, if not cached, it calls the original `FindManifest`. The old `FindManifest` finds the manifest from storage if passed with a rdg file name; or the latest version manifest in a directory if a rdg directory is provided. Since the original `FindManifest` does not contain `latest` in its name, I don't think the new `FindManifest` needs it. If we really want a new name, I think `FindManifestCached` is better, but since it also finds the manifest from storage if not cached, I feel `FindManifest` with different signature should be fine.